packages/volto-hydra/src/components/Iframe/View.jsx

```diff
@@ -51,15 +47,25 @@ const Iframe = () => {
     });
   }, [token]);
 
+  useEffect(() => {
+    if (typeof window !== 'undefined' && Object.keys(form).length > 0) {
```

Comment:
Should be any need to check for window here, we're in a useEffect which is always client-side